ui/src/components/ConfigModalContent/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { Modal, Heading4, TextLink } from "@stellar/design-system";
+import { Json } from "basics/Json";
+
+export const ConfigModalContent: React.FC = () => (
+  <>
+    <Modal.Heading>Configuration Files</Modal.Heading>
+    <Modal.Body>
+      <p>
+        Configuration files are required for running SEP-6, 12, and 31 tests. 
+        These files provide information required for the test suite to interact with your anchor service properly.
+      </p>
+      <p>
+        Configuration files are JSON-formatted and have top-level keys for each SEP for which tests are run.
+      </p>
+      <Json src={{
+        "6": {},
+        "12": {},
+        "31": {}
+      }}></Json>
+      <p>
+        The formats for each SEP are outlined below.
+      </p>
+
+      <Heading4>SEP-6</Heading4>
+      <p>
+        SEP-6 configuration objects are used for specifying the URL parameters that should be included to GET /deposit and GET /withdraw requests.
+      </p>
+      <Json src={{
+        "deposit": {
+          "transactionFields": {}
+        },
+        "withdraw": {
+          "types": {
+            "bank_account (example)": {
+              "transactionFields": {}
+            }
+          }
+        }
+      }}></Json>
+      <ul>
+        <li>
+          <strong>deposit</strong>: contains a single "transactionFields" key
+          <ul>
+            <li><strong>transactionFields</strong>: contains the key-value pairs to use as parameters to GET /deposit requests. These items should correspond directly to the items outlined in GET /info's <TextLink href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#for-each-deposit-asset-response-contains">fields</TextLink> object</li>
```

Comment:
this is super NIT: but can you replace the `'` in the copy with an apostrophe `’`

It's part of the product style guide. I'll do a pass over the whole UI to fix instances where we didn't catch this

https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#use-the-right-symbols